docs/release-notes/release-notes-0.13.995.md

```diff
@@ -18,13 +17,15 @@
 
 ### Taproot Assets
 
-* A few bugs related to TAP channel liquidity were fixed, sending very small or
-very big asset amounts is now possible. See [tapd PR](https://github.com/lightninglabs/taproot-assets/pull/1120).
-* `ListBalances` now supports the `include_leased` flag, which will include leased
-asset balances in the response.
+- A few bugs related to TAP channel liquidity were fixed, sending very small or very big
+  asset amounts is now possible. See
+  [tapd PR](https://github.com/lightninglabs/taproot-assets/pull/1120).
+- `ListBalances` now supports the `include_leased` flag, which will include leased asset
```

Comment:
I think we generally prefer the `*` character for list entries.

Reply:
Here as well. :cry: